frontend/src/app/components/wp-details/wp-details-toolbar.component.ts

```diff
@@ -28,7 +28,7 @@
 
 import {HalResourceEditingService} from "core-app/modules/fields/edit/services/hal-resource-editing.service";
 import {I18nService} from 'core-app/modules/common/i18n/i18n.service';
-import {Component, Inject, Input} from '@angular/core';
+import {Component, Input} from '@angular/core';
 import {WorkPackageResource} from 'core-app/modules/hal/resources/work-package-resource';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/c9bc013cc27eb3bb23aea9c85aaedb5b868ef858/frontend/src/app/components/wp-details/wp-details-toolbar.component.ts#L33"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>